python/pyarrow/lib.pxd

```diff
@@ -209,6 +214,18 @@ cdef class LargeListValue(ArrayValue):
     cdef int64_t length(self)
 
 
+cdef class MapValue(ArrayValue):
+    cdef readonly:
+        DataType key_type
+        DataType item_type
+
+    cdef:
+        CMapArray* ap
+
+    cdef getitem(self, int64_t i)
+    cdef int64_t length(self)
+
+
```

Comment:
Need to add MapArray as well in this file? (a bit below, around line 368)

Reply:
Yeah, looks like it's like a forward declaration? I added it but not sure why it's needed..